tools/mo/openvino/tools/mo/front/tf/loader.py

```diff
@@ -10,6 +10,7 @@
 from openvino.tools.mo.utils.error import Error, FrameworkError
 from openvino.tools.mo.utils.utils import refer_to_faq_msg
 from openvino.tools.mo.utils.versions_checker import get_environment_setup
+from tensorflow.python.tools import saved_model_utils
```

Comment:
it looks nowhere used routine. Please remove

Reply:
Done.

Reply:
Done.